app/Http/Controllers/RepositoryController.php

```diff
@@ -39,4 +39,20 @@ public function store(StoreRepositoryRequest $request): RedirectResponse
 
         return to_route('repositories.index');
     }
+
+    /**
+     * Remove the specified resource.
+     *
+     * @param Repository $repository
+     *
+     * @return RedirectResponse
+     */
+    public function destroy(Repository $repository): RedirectResponse
+    {
+        abort_if($repository->user_id !== auth()->id(), 403);
```

Comment:
please use the form request and authorize section instead of doing this in the controller.

Reply:
As I said [here](https://github.com/geeksesi/code-review-pals/pull/68#discussion_r1168713917) I don't agree with that on a project of this size.

@vassilidev can you give your opinion?

Reply:
use a policy and replace to $this->authorize('action', model::class);